dist/.github/workflows/test.yml

```diff
@@ -37,6 +47,44 @@ jobs:
           # Pre-releases
           - otp-version: '23.0'
             elixir-version: '1.11.0-rc.0'
+          # Fallback to Elixir (with no OTP)
+          - otp-version: '24.0-rc1'
+            elixir-version: '1.9.3'
+          # Miscellaneous combos
+          - elixir-version: 'master'
+            otp-version: 'OTP-22.3'
+          - elixir-version: 'master'
+            otp-version: 'OTP-22'
+          - elixir-version: 'master'
+            otp-version: '^22'
+          - elixir-version: 'master-otp-22'
+            otp-version: 'OTP-22'
+          - elixir-version: '1.6.5'
+            otp-version: 'maint-21'
+          - elixir-version: 'v1.6.5'
+            otp-version: 'maint-21'
+          - elixir-version: 'v1.9.x'
+            otp-version: '21.1'
+          - elixir-version: '1.9.x'
+            otp-version: '21'
+          - elixir-version: 'v1.8.x'
+            otp-version: 'OTP-22'
+          - elixir-version: 'v1.8.x-otp-23'
+            otp-version: '23.1'
+          - elixir-version: 'v1.8-otp-21'
+            otp-version: '^22.3'
+          - elixir-version: 'v1.10-otp-21'
+            otp-version: 'OTP-21.x'
+          - elixir-version: 'v1.x'
+            otp-version: 'master'
+          - elixir-version: 'v1.6.x-otp-20'
+            otp-version: '21.1'
+          - elixir-version: 'v1.11.0-rc.0-otp-23'
```

Comment:
This is the only part I don't understand. I'm not sure this was supported before, but I don't have strong feelings about either.  It may be best if we don't document this, that said, but yeah, no strong feelings. Am fine with it myself

Reply:
We don't need to document it. It actually isn't in the `README.md`. I added it to/kept it in the tests because I expect it to work 😄 , but I can take it out if you prefer.

I also prefer (and that's what we recommend) to use simple versions like 1.0.0, for `rebar3`, Elixir and Erlang/OTP, and those, too, will work.

Are you specifically talking about the `rc-` part of the `otp-` part? Because for `rc-` I see real value; I want to be able to preemptively test my builds with e.g. 24.0-rc2 (out today 🎉).

Reply:
otp part, not the rc part, indeed.

Reply:
Yeah. I can remove the `-otp-` if you prefer, but I imagine it doesn't hurt. @ericmj?

Reply:
I would say remove it because there should be no reason to set it manually.

Reply:
I'll make a separate commit to remove the possibility to have `OTP-` as a prefix in Erlang/OTP versions, too. This seems consistent with your request. If it's not something you guys prefer I should easily be able to rollback.